### DISCUSSION TITLE
GDAL Versions used in GemGIS

### DISCUSSION THREAD

**Discussion by RichardScottOZ** [cgre-aachen/gemgis#173] [Installation] [answered]
I noticed that following the installation order - rasterio downgrades gdal from 3.2.2 to 3.1.4.

**Comment by RichardScottOZ** [comment]
Also in installation page python 3.8.5 and 3.9 in readme

**Reply by AlexanderJuestel** [reply]
Yes, that is something we should be more consistent with, actually.

**Comment by RichardScottOZ** [comment]
so do you prefer 3.8 or 3.9 currently?

**Reply by AlexanderJuestel** [reply]
I have no difference to be honest. Both versions work for me

**Comment by RichardScottOZ** [comment, marked as answer]
Thanks - made one of each but have been using the 3.8 one.
